### ISSUE TITLE
HTML Report: Orange failure blocks cannot be reached with keyboard

### ISSUE THREAD

**Issue by jernst** [fediverse-devnet/feditest#415]
From Accessibility Report:

> You can’t reach the orange failure blocks when using the keyboard. By using ‘tab’ on your keyboard, you should be able to cycle through all interactive elements on the page.

References:

* https://www.w3.org/WAI/WCAG22/Understanding/keyboard
* https://www.w3.org/WAI/WCAG22/Understanding/focus-order
* https://www.w3.org/WAI/WCAG22/Understanding/name-role-value

**Comment by jernst** [comment]
Refiled as #435.
